src/Uno.Extensions.Reactive.Tests/Messaging/Given_Messaging.cs

```diff
@@ -44,6 +44,93 @@ public async Task When_Updated_Then_StateUpdated()
 		result.Should().BeEquivalentTo(new MyEntity(42, 1));
 	}
 
+	[TestMethod]
+	public async Task When_Fluent_Value_Updated_Then_StateUpdated()
+	{
+		var messenger = new WeakReferenceMessenger();
+		var state = State.Value(this, () => new MyEntity(42))
+						 .Observe(messenger, i => i.Key);
```

Comment:
Same here about styling, we recommend to break before each operator, including the first one:

```suggestion
		var state = State
			.Value(this, () => new MyEntity(42))
			.Observe(messenger, i => i.Key);
```